projects/netgrif-components/src/lib/data-fields/process-ref/process-ref-default/process-ref-default.component.scss

```
@@ -0,0 +1,14 @@
+.builder-height {
+    width: 100%;
+    max-width: 100%;
+    display: flex;
+    overflow: hidden;
+    contain: size;
+    border: 0;
+
+    ::ng-deep > * {
```

Stylelint (17.14.1)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)